app/services/deployment/server_actions/create.rb

```diff
@@ -28,10 +28,11 @@ def deploy_configuration(configuration, state)
 
       def create_server(configuration, state)
         server = ServerAccess::Heroku.new(name: configuration.application_name)
-        state
-          .add_state(:create_server) { server.create }
-          .add_state(:build_addons) { server.build_addons(configuration.addons) }
-          .add_state(:update_env_variables) { server.update_env_variables(configuration.env_variables) }
+        Deployment::Helpers::AddonsBuilder.new(
+          configuration,
+          state.add_state(:create_server) { server.create },
```

Comment:
please perform it before calling `AddonsBuilder`

Reply:
Will do